### PR TITLE
Release 1.19.0-alpha.3

### PR DESCRIPTION
**Pull request by hakman**
/cc @justinsb @rifelpet @johngmyers @geojaz @kashifsaadat @mikesplain @rdrgmnzs @zetaab 
/hold for approvals

### CODE REVIEW

**Review by zetaab — APPROVED** [review]
/lgtm

File: hack/verify-gomod.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ cd "${KOPS_ROOT}"
 
 make gomod
 
-changes=$(git status --porcelain || true)
+changes=$(git status --porcelain vendor/ || true)
```
Comment (johngmyers) [inline hack/verify-gomod.sh:27]: I think this might remove the assertion that go.mod/go.sum are tidy. It also removes the (redundant) bazel assertion.

Comment (hakman) [reply hack/verify-gomod.sh]: Good catch. Updated.

**Comment by hakman** [comment]
Thanks everyone 👍 
/hold cancel

**Pull request merged**